### PR TITLE
Docs 4720 explain certificates

### PR DESCRIPTION
**Pull request by valkyrie69**
Trying to add more description of the certificate that you add to a secret group.

### CODE REVIEW

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 1]
```diff
@@ -23,44 +23,31 @@ The name of your secret group must:
 
 The secret group appears in the *Secret Groups* list view. Click *Edit* to add the necessary secret types, for example, a keystore or a truststore. 
 
-== Add a Keystore
+[[add_certificate]]
+== Add a Certificate
 
-Specify the type of keystore to add to the secret group. 
+Add a single PEM or JKS certificate to the secret group. This certificate is the "leaf" certificate (end-entity certificate).
```
Comment (hegderohit89) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: You may want to remove "JKS"

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 2]
```diff
@@ -23,44 +23,31 @@ The name of your secret group must:
...
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+** PEM
+* *Certificate File* -  Click *Choose File*, and select the certificate file to upload.
+* *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
 . Click *Save*.
 
 == Add a Truststore
 
-Add a truststore for public certificates of trusted servers. 
+Add a truststore for public certificates of trusted servers. The truststore is used to store certificates from the `trusted` CA, which are then used to verify certificates presented by the server in an SSL connection.
```
Comment (hegderohit89) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Do we want to highlight the `trusted` part ?

**Review by hegderohit89 — APPROVED** [review]

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 1 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: good catch!

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 3]
```diff
@@ -23,44 +23,31 @@ The name of your secret group must:
...
 +
-** *Keystore File* - Click *Choose File* to locate and upload the keystore file to use.
-** *Keystore Passphrase* - Enter the word or phrase that protects the keystore.
-** *Alias* - The alias used to access the keystore entries (key and trusted certificate entries).
-** *Key Passphrase* - The word or phrase that protects the private key.
-** *Algorithm* (Optional) - The algorithm to use for encryption of keys.
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+** PEM
+* *Certificate File* -  Click *Choose File*, and select the certificate file to upload.
+* *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:42]: I don't understand the bullet nesting here---should cert file and override exp date be at the same level as PEM? Are they part of a different list? Maybe PEM should be one bullet not two?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 4]
```diff
@@ -80,24 +67,38 @@ Supported types are:
...
-. Select *Certificate* in the menu on the left, and click *Add Certificate*. 
-. In the *Add Certificate* screen, add the required information. 
+. In the *Secret Groups* list view, select the secret group to which to add a keystore. 
+. Select *Keystore* in the menu on the left, and click *Add Keystore*. 
+. In the *Name* field, enter a name for your keystore.
+. In the *Type* field, select the keystore type from the drop-down menu.
 +
-* *Name* - Enter a name for your certificate.
-* *Type* -  Select the certificate type from the drop-down menu.
+Supported types include the following:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: you don't need "the following" (fine to keep for consistency with other sections or topics).

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 5]
```diff
@@ -80,24 +67,38 @@ Supported types are:
...
 +
-Supported types are:
+* Privacy Enhanced Mail (PEM) - Base64 encoded ASCII file with a `cer`, `crt`, or `pem` extension.
+* Java Keystore (JKS) - Repository for authorization or public key certificates. The JKS keystore type does not store secret keys.
+* PKCS12 - Stores server and intermediate certificates in an archive file format. The PKCS12 keystore type does not store secret keys.
+* Java Cryptography Extension KeyStore (JCEKS) - Stores server and intermediate certificates as well as secret keys.
+.. To add a *PEM* type keystore, you must provide the following:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Should this be under the PEM bullet line 82?

**Review by Mulesoft-Mysti — APPROVED** [review]

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 3 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc:42]: Verified the bullet nesting, and it's as intended.

**Pull request merged**